pages/registration.tsx

```diff
@@ -0,0 +1,24 @@
+import Head from "next/head";
```

Comment:
`registration.tsx` -> `register.tsx` - as per the navbar.

Reply:
Do you think "Registration" is semantically better as the navigation word is "Registrasi"?

Reply:
@adityapurwa Hmm, you're right. Feel free to pick whichever you like, but please update the navbar items accordingly.